maintenance_curation.Rmd

```diff
@@ -0,0 +1,229 @@
+# Package Curation Policy {#curationpolicy}
+
+```{block, type='summaryblock'}
+This chapter summarizes a proposed curation policy for rOpenSci's
+ongoing maintenance of packages developed as part of rOpenSci activities
+and/or under the rOpenSci GitHub organization. This curation policy aims
+to support these goals:
+
+-   Ensure packages provided by rOpenSci are up-to-date and high quality
+
+-   Provide clarity as to the development status and and review status
+    of any software in rOpenSci repositories
+
+-   Manage maintenance effort for rOpenSci staff, package authors, and
+    volunteer contributors
+
+-   Provide a mechanism to gracefully sunset packages while maintaining
+    peer-review badging
+
+```
+
+Elements of infrastructure described
+below needed for implementation of the policy are in some cases partly
+built and in other cases not yet begun. We aim to adopt this policy in
+part to prioritize work on these components.
+
+## The package registry
+
+-   The rOpenSci package
+[registry](https://github.com/ropensci/roregistry)
+is a central listing of R packages currently or formerly
+maintained or reviewed by rOpenSci. It contains essential package
+metadata including development and review status, and will be the
+source of data for display on websites, badges, etc. It will allow
+this listing to be maintained independently of package or
+infrastructure hosting platforms.
+
+## Staff-maintained packages
+
+Staff-maintained packages are developed and maintained by rOpenSci staff
+as part of rOpenSci projects. These packages may also be peer-reviewed
+packages, but are not necessarily peer reviewed. Many are infrastructure
+packages that fall out of scope for peer review.
+
+-   Staff-maintained packages will be listed in the registry with tag
+    "staff" and listed on rOpenSci's packages web page or similar
+    venues with tag "staff"
+
+-   These packages will be stored in the "ropensci" GitHub
+    organization
+
+-   Staff-maintained packages will be checked via rOpenSci's [custom CI
+    system](https://status.ropensci.org/) to test that they build and test successfully across
+    systems and that their documentation build correctly.
+
+-   When the packages fail checks, rOpenSci staff will endeavour to fix
+    changes, prioritizing packages based on user base (downloads),
+    reverse dependencies, or strategic goals.
+
+-   On a biannual or annual basis, rOpenSci will review all packages
+    that have been failing for over a month to determine whether to transfer them to the ["ropensci-archive" GitHub organization](https://github.com/ropensci-archive).
```

Comment:
If I may chime in: 

It would be super useful to have a (very) cursory explanation as to why these package have been archived exactly. I have noticed some of these packages over the last couple of weeks and I was left wondering if the upstream API shut down (in the case of API package), if the package was superseded by another one or if it was a lack of interest/time from the current maintainer (probably not the case since a call for a new maintainer would have be made???).

All of this could help find someone to take over in the appropriate cases.

On the other hand, I understand this would increase the workload of maintainers and/or rOpenSci staff :thinking:

Reply:
thanks for the suggestion, good idea! Would you expect this information to be in the README of the archived package?

Reply:
Yes, that would be the first place I check